lib.php

```diff
@@ -45,18 +45,29 @@
  */
 function hvp_supports($feature) {
     switch($feature) {
-        case FEATURE_GROUPS:                  return true;
-        case FEATURE_GROUPINGS:               return true;
-        case FEATURE_GROUPMEMBERSONLY:        return true;
-        case FEATURE_MOD_INTRO:               return false;
-        case FEATURE_COMPLETION_TRACKS_VIEWS: return false;
-        case FEATURE_COMPLETION_HAS_RULES:    return false;
-        case FEATURE_GRADE_HAS_GRADE:         return true;
-        case FEATURE_GRADE_OUTCOMES:          return false;
-        case FEATURE_BACKUP_MOODLE2:          return false;
-        case FEATURE_SHOW_DESCRIPTION:        return false;
-
-        default: return null;
+        case FEATURE_GROUPS:
+return true;
+        case FEATURE_GROUPINGS:
+return true;
+        case FEATURE_GROUPMEMBERSONLY:
+return true;
+        case FEATURE_MOD_INTRO:
+return false;
+        case FEATURE_COMPLETION_TRACKS_VIEWS:
+return false;
+        case FEATURE_COMPLETION_HAS_RULES:
+return false;
+        case FEATURE_GRADE_HAS_GRADE:
+return true;
+        case FEATURE_GRADE_OUTCOMES:
+return false;
+        case FEATURE_BACKUP_MOODLE2:
+return false;
+        case FEATURE_SHOW_DESCRIPTION:
+return false;
+
+        default:
+return null;
```

Comment:
Did something go wrong here? It seems a bit strange that there should be no indentation.